### PR TITLE
Cybersecurity New Topic: Cyber Resilience

### PR DESCRIPTION
**Pull request by jonchery**
### Description

Added a new cybersecurity folder and concept, Cyber Resilience

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Cyber Resilience'
+Description: 'Cyber Resilience is the ability of an organization to withstand and recover from cyber incidents.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+CatalogContent:
+  - 'paths/cyber-resilience-risk-management'
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:9]:
```suggestion
  - 'paths/cyber-resilience-risk-management'
  - 'paths/fundamentals-of-cybersecurity'
```
Generally we want two slugs under CatalogContent

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Cyber Resilience'
+Description: 'Cyber Resilience is the ability of an organization to withstand and recover from cyber incidents.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:7]:
```suggestion
  - 'Cybersecurity'
  - 'Infrastructure'
```
We generally want more than one tag.

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Cyber Resilience In-Use:
+
+Becoming cyber resilient is essential to preventing business disruptions that may result from a cybersecurity incident. In order to become cyber resilient, an organization must first consider its current state security posture. Most importantly, the so-called crown jewels (i.e. critical business/service infrastructure) must be identified, prioritized, and protected. Actions such as application control and implementing the principle of least privilege can lessen risk and increase resilience.
+
+## Ways to Achieve Cyber Resilience
+
+In Cybersecurity, some concepts that go into achieving Cyber Resiliency are:
+
+- _Redundancy_ -> In Cybersecurity, redundancy refers to having alternate ways to maintain the availability of a resource.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:22]:
```suggestion
- **Redundancy:** Refers to having alternate ways to maintain the availability of a resource.
```

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- _Replication_ -> In Cybersecurity, replication refers to having multiple copies of the same data available in multiple locations.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:24]:
```suggestion
- **Replication:** Refers to having multiple copies of the same data available in multiple locations.
```

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- _Non-Persistence_ -> This concept refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:26]:
```suggestion
- **Non-Persistence:** Refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers. 
```

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+In Cybersecurity, some concepts that go into achieving Cyber Resiliency are:
+
+- _Redundancy_ -> In Cybersecurity, redundancy refers to having alternate ways to maintain the availability of a resource.
+
+- _Replication_ -> In Cybersecurity, replication refers to having multiple copies of the same data available in multiple locations.
+
+- _Non-Persistence_ -> This concept refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers. 
+
+- _High Availability_ ->  Refers to the assurance that a system will be available to respond to requests and complete requested operations in a timely manner.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:28]:
```suggestion
- **High Availability:** Refers to the assurance that a system will be available to respond to requests and complete requested operations in a timely manner.
```

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+- _Restoration Order_ -> Refers to the sequence of mission-critical business processes that should be restored.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:30]:
```suggestion
- **Restoration Order:** Refers to the sequence of mission-critical business processes that should be restored. 
```

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+- _Replication_ -> In Cybersecurity, replication refers to having multiple copies of the same data available in multiple locations.
+
+- _Non-Persistence_ -> This concept refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers. 
+
+- _High Availability_ ->  Refers to the assurance that a system will be available to respond to requests and complete requested operations in a timely manner.
+
+- _Restoration Order_ -> Refers to the sequence of mission-critical business processes that should be restored. 
+
+- _Diversity_ -> Refers to the multilayered security mechanism of having different types of access control provided by different entities/products.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:32]:
```suggestion
- **Diversity:** Refers to the multilayered security mechanism of having different types of access control provided by different entities/products.
```

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+**Cyber Resilience** is the ability of an organization to withstand and recover from cyber incidents. It describes the ability to continue operating despite experiencing challenging cyber events such as ransomware, Distributed Denial of Service (DDoS), social engineering, and insider threats.
+
+## Cyber Resilience In-Use:
+
+Becoming cyber resilient is essential to preventing business disruptions that may result from a cybersecurity incident. In order to become cyber resilient, an organization must first consider its current state security posture. Most importantly, the so-called crown jewels (i.e. critical business/service infrastructure) must be identified, prioritized, and protected. Actions such as application control and implementing the principle of least privilege can lessen risk and increase resilience.
+
+## Ways to Achieve Cyber Resilience
+
+In Cybersecurity, some concepts that go into achieving Cyber Resiliency are:
+
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:21]: In addition to the edits below, remove the line breaks between each bullet item.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jonchery I did a first review and left a few comments, most about formatting your bulleted list.

**Comment by SSwiniarski** [comment]
@jonchery, one more edit. Can we remove the trailing punctuation from the headers? The colon `:` will cause this to fail one of the checks.

**Review by jonchery — COMMENTED** [review]
Removing ":"

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+## Cyber Resilience In-Use
+
+Becoming cyber resilient is essential to preventing business disruptions that may result from a cybersecurity incident. In order to become cyber resilient, an organization must first consider its current state security posture. Most importantly, the so-called crown jewels (i.e. critical business/service infrastructure) must be identified, prioritized, and protected. Actions such as application control and implementing the principle of least privilege can lessen risk and increase resilience.
+
+## Ways to Achieve Cyber Resilience
+
+In Cybersecurity, some concepts that go into achieving Cyber Resiliency are:
+- **Redundancy:** Refers to having alternate ways to maintain the availability of a resource.
+- **Replication:** Refers to having multiple copies of the same data available in multiple locations.
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:24]: I think this definition should be tweaked a bit to differentiate it from redundancy. I see redundancy as having multiple copies or alternative ways to provide some service. I would expect that replication is a more active process, or set of systems to reproduce data in the event of loss or failure.

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+- **Non-Persistence:** Refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers.
```
Comment (caupolicandiaz) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:25]:
```suggestion
- Non-Persistence: Refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers. 
```

Comment (caupolicandiaz) [reply content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:25]: I don't think this actually gets at the core of non-persistence. I understand that this strategy is about limiting the availability of access, or data, etc. I don't think that idea is represented clearly in this definition.

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+Becoming cyber resilient is essential to preventing business disruptions that may result from a cybersecurity incident. In order to become cyber resilient, an organization must first consider its current state security posture. Most importantly, the so-called crown jewels (i.e. critical business/service infrastructure) must be identified, prioritized, and protected. Actions such as application control and implementing the principle of least privilege can lessen risk and increase resilience.
+
+## Ways to Achieve Cyber Resilience
+
+In Cybersecurity, some concepts that go into achieving Cyber Resiliency are:
+- **Redundancy:** Refers to having alternate ways to maintain the availability of a resource.
+- **Replication:** Refers to having multiple copies of the same data available in multiple locations.
+- **Non-Persistence:** Refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers. 
+- **High Availability:** Refers to the assurance that a system will be available to respond to requests and complete requested operations in a timely manner.
```
Comment (caupolicandiaz) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:26]:
```suggestion
- High Availability: Refers to the assurance that a system will be available to respond to requests and complete requested operations in a timely manner.
```

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+- **Restoration Order:** Refers to the sequence of mission-critical business processes that should be restored.
```
Comment (caupolicandiaz) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:27]:
```suggestion
- Restoration Order: Refers to the sequence of mission-critical business processes that should be restored. 
```

File: content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+
+## Ways to Achieve Cyber Resilience
+
+In Cybersecurity, some concepts that go into achieving Cyber Resiliency are:
+- **Redundancy:** Refers to having alternate ways to maintain the availability of a resource.
+- **Replication:** Refers to having multiple copies of the same data available in multiple locations.
+- **Non-Persistence:** Refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers. 
+- **High Availability:** Refers to the assurance that a system will be available to respond to requests and complete requested operations in a timely manner.
+- **Restoration Order:** Refers to the sequence of mission-critical business processes that should be restored. 
+- **Diversity:** Refers to the multilayered security mechanism of having different types of access control provided by different entities/products.
```
Comment (caupolicandiaz) [inline content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md:28]:
```suggestion
- Diversity: Refers to the multilayered security mechanism of having different types of access control provided by different entities/products.
```

**Review by caupolicandiaz — COMMENTED** [review]
@jonchery I've completed my review, please update and tag me when you've finished making changes. Thanks

**Comment by caupolicandiaz** [comment]
@jonchery This entry has been sitting for quite some time. Please advise if you can't continue to work on this contribution.

**Comment by jonchery** [comment]
> caupolicandiaz

This got lost in my spam. Resolved the issues.

**Review by caupolicandiaz — APPROVED** [review]
@jonchery Looks good.

**Pull request merged**